### PR TITLE
Translate tutorial/part-six

### PR DESCRIPTION
**Pull request by JubayerJoy**
Signed-off-by: Jubayer Abdullah Joy <jubayerjoy98@gmail.com>

### Current State
- [x] Translation of `tutorial/part-six` complete 
#### Work in Progress
- [x] Review
### Note for Reviewers
- Translation is complete, kept as close to original and same ammount of lines
Please Review 🚀

### CODE REVIEW

**Comment by JubayerJoy** [comment]
@nutboltu, @shakib609 please review. Thanks! 🚀

File: docs/tutorial/part-six/index.md [hunk 1]
```diff
@@ -190,35 +190,35 @@ seem to really enjoy bananas!
...
 
-Try creating a new page containing a blog post and see what happens to the list of blog posts on the homepage!
+একটা নতুন ব্লগপোস্ট পেইজ তৈরি করার চেস্টা করুন এবং হোম পেইজের ব্লগ পোস্টের লিস্টে কি পরিবর্তন হয় দেখুন!
 
-## What's coming next?
+## এরপর কি আসছে?
 
-This is great! You've just created a nice index page where you're querying your markdown
-files and producing a list of blog post titles and excerpts. But you don't want to just see excerpts, you want actual pages for your markdown files.
+অনেক দারুণ লাগছে! আপনি মাত্রই একটা Index পেইজ তৈরি করেছেন যেখানে আপনি মার্কডাউন ফাইল্গুলো
```
Comment (nutboltu) [inline docs/tutorial/part-six/index.md]: ফাইল্গুলো -> ফাইলগুলো

File: docs/tutorial/part-six/index.md [hunk 2]
```diff
@@ -190,35 +190,35 @@ seem to really enjoy bananas!
...
+Query করছেন এবং একটি ব্লগপোস্টের টাইটেল এবং উদ্ধৃতাংশ (Excerpt) এর লিস্ট তৈরি করছেন। কিন্তু আপনি শুধু উদ্ধৃতাংশ চান না, আপনি মার্কডাউন ফাইলের আসল পেইজ চান।
```
Comment (nutboltu) [inline docs/tutorial/part-six/index.md]: Let's remove this extra (Excerpt)

**Review by nutboltu — CHANGES_REQUESTED** [review]
LGTM. minor changes requested.

File: docs/tutorial/part-six/index.md (hunk 1 above)
Comment (JubayerJoy) [reply docs/tutorial/part-six/index.md]: Very good spot! Fixed, Thank you. 🎈

File: docs/tutorial/part-six/index.md (hunk 2 above)
Comment (JubayerJoy) [reply docs/tutorial/part-six/index.md]: I was thinking the same, removed it vai! 👍

**Review by nutboltu — APPROVED** [review]
awesome! @JubayerJoy

**Pull request merged**